core/src/test/kotlin/io/specmatic/core/SpecmaticConfigKtTest.kt

```diff
@@ -183,7 +188,8 @@ internal class SpecmaticConfigKtTest {
         val props = mapOf(
             SPECMATIC_GENERATIVE_TESTS to "false",
             VALIDATE_RESPONSE_VALUE to "false",
-            EXTENSIBLE_SCHEMA to "false"
+            EXTENSIBLE_SCHEMA to "false",
+            EXAMPLE_DIRECTORIES to "api/tests,api/stubs"
```

Comment:
This was initially confusing as we typically won't pass test and stub directories together. This should have either two test directoreis or two stub directores.

Reply:
Thanks, I've renamed these folders to be generic to avoid this confusion.